doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,16 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]Example:[/b]
+		[codeblock]
+		var spin_box = SpinBox.new()
+		add_child(spin_box)
+		var line_edit = SpinBox.get_line_edit()
```

Comment:
Should be `spin_box.get_line_edit()`. You have to get the LineEdit on the SpinBox *instance*, which is referenced by your `spin_box` variable. If you try the above code, you'll see that it raises an error.

Reply:
missed that :flushed:!